### PR TITLE
Add stubs for some exercises #735

### PR DESCRIPTION
**Pull request by hyuko21**
I've added stubs for **zipper**, **wordy** and **word-search** listed at  #705

### CODE REVIEW

File: exercises/word-search/word-search.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  constructor() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  find() {
+    throw new Error("Remove this statement and implement this function");
+  }
+}
+
+module.exports = WordSearch;
```
Comment (SleeplessByte) [inline exercises/word-search/word-search.js]: If you look at the examples and other stubs, you see that we use `es6` `export`, not `CommonJS` `module.exports`.

File: exercises/word-search/word-search.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+//
+// This is only a SKELETON file for the 'Word Search' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+function searchHorizontally() {
```
Comment (SleeplessByte) [inline exercises/word-search/word-search.js]: All these top-level functions should _not_ be stubbed, as an implementation for this exercises does not necessarily need them. The `class` and `find` are correct.

Comment (tejasbubane) [reply exercises/word-search/word-search.js]: You can check corresponding [spec file](https://github.com/exercism/javascript/blob/4a0f58d67f27fda8d433f934aaeef386f3f1033e/exercises/word-search/word-search.spec.js) for the exercise and stub **only** the functions/classes used there.

Comment (hyuko21) [reply exercises/word-search/word-search.js]: Thanks for your replies. Sorry for those mistakes, I didn't realize it at first.

I've had made those requested changes.

**Comment by SleeplessByte** [comment]
Great fix @hyuko21 , thanks!!

**Pull request merged**